library/core/functions.general.php

```diff
@@ -3616,18 +3610,6 @@ function trace($Value = null, $Type = TRACE_INFO) {
     }
 }
 
-if (!function_exists('trueStripSlashes')) {
```

Comment:
You don't want to remove this function at this point. Just make a call to `deprecated()`.